schemas/app-sre/integration-1.yml

```diff
@@ -58,6 +58,8 @@ properties:
       check_only_affected_shards:
         type: boolean
         description: "defaults to false"
+      run_order:
```

Comment:
Is it possible to restrict range? To match supported implementation.

Reply:
I see, current implementation does not restrict the numbers. Guess this is not required then